Procfile

```diff
@@ -1,3 +1,3 @@
 web: bundle exec puma -C config/puma.rb
+clock: bundle exec clockwork config/clock.rb
```

Comment:
ak nezalezi na poradi tak to drzme abecedne: clock, web, worker

Reply:
myslim ze nezalezi. motivacia bola nech to je jednotne s ostatnymi projektami, kde je to web-clock-worker. tak pomenime to vsade na abecedne?

Reply:
abecedne